db/migrate/20220511142620_add_id_to_income.rb

```diff
@@ -1,9 +0,0 @@
-# frozen_string_literal: true
```

Comment:
Is it correct to delete migration? Possible we should create new?

Reply:
No, a migration cannot be deleted. Need to create a new migration that removes the columns which need.